### PR TITLE
HDDS-4582 Update `quota in counts` to `quota in namespace`

### PR DESCRIPTION
**Pull request by amaliujia**
## What changes were proposed in this pull request?

Use quota in namespace to improve readability of variables that are related to namespace quota.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4582

## How was this patch tested?

UT

### CODE REVIEW

**Comment by amaliujia** [comment]
R: @linyiqun

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/SetSpaceQuotaOptions.java [hunk 1]
```diff
@@ -32,14 +32,14 @@
   @CommandLine.Option(names = {"--count-quota"},
       description = "For volume this parameter represents the number of " +
```
Comment (linyiqun) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/SetSpaceQuotaOptions.java:33]: Can you also rename '--count-quota' to '--namespace-quota'?

File: hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell-lib.robot [hunk 2]
```diff
@@ -48,12 +48,12 @@ Test ozone shell
                     Should Be Equal     ${result}       DISK
     ${result} =     Execute             ozone sh bucket info ${protocol}${server}/${volume}/bb1 | jq -r '. | select(.name=="bb1") | .quotaInBytes'
                     Should Be Equal     ${result}       10995116277760
-    ${result} =     Execute             ozone sh bucket info ${protocol}${server}/${volume}/bb1 | jq -r '. | select(.name=="bb1") | .quotaInCounts'
+    ${result} =     Execute             ozone sh bucket info ${protocol}${server}/${volume}/bb1 | jq -r '. | select(.name=="bb1") | .quotaInNamespace'
                     Should Be Equal     ${result}       100
                     Execute             ozone sh bucket setquota ${protocol}${server}/${volume}/bb1 --space-quota 1TB --count-quota 1000
```
Comment (linyiqun) [inline hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell-lib.robot:53]: Once option --count-quota is updated to --namespace-quota, this file also needed to be updated.

**Review by linyiqun — COMMENTED** [review]
Thanks for fixing this, @amaliujia .
Minor comments below.

**Comment by amaliujia** [comment]
Also R @captainzmc

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/SetSpaceQuotaOptions.java (hunk 1 above)
Comment (amaliujia) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/SetSpaceQuotaOptions.java:33]: Good point!

File: hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell-lib.robot (hunk 2 above)
Comment (amaliujia) [reply hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell-lib.robot]: Done!

**Review by linyiqun — APPROVED** [review]
LGTM, +1.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java [hunk 3]
```diff
@@ -135,11 +135,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       } else {
         omVolumeArgs.setQuotaInBytes(omVolumeArgs.getQuotaInBytes());
       }
-      if (checkQuotaCountsValid(setVolumePropertyRequest.getQuotaInCounts())) {
-        omVolumeArgs.setQuotaInCounts(
-            setVolumePropertyRequest.getQuotaInCounts());
+      if (checkQuotaCountsValid(
```
Comment (captainzmc) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java:138]: Do we also change checkQuotaCountsValid to checkQuotaNamespaceValid?

**Review by captainzmc — COMMENTED** [review]
LGTM over all. Just one comment about method naming.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java (hunk 3 above)
Comment (amaliujia) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java]: Indeed! Done!

**Review by captainzmc — APPROVED** [review]
LGTM, +1.

**Comment by captainzmc** [comment]
Merged this. Thanks for @amaliujia's patch and @linyiqun's review.

**Pull request merged**